src/components/K8s.vue

```diff
@@ -24,7 +32,10 @@
 </template>
 
 <script>
-import Checkbox from './Checkbox.vue';
+import Checkbox from '@/src/components/Checkbox.vue';
+import SystemPreferences from "@/src/components/SystemPreferences.vue";
```

Comment:
Bike shedding: would `VMPreferences` be a better name? Then we can stuff any configuration about virtual machines there.

Reply:
we'll get there as we refined the design